{{ cookiecutter.app_name }}/src/{{ cookiecutter.module_name }}/app.py

```diff
@@ -115,6 +115,54 @@ def main():
         starting_scene={{ cookiecutter.class_name }},
         title=metadata['Formal-Name'],
     )
+{%- elif cookiecutter.gui_framework == 'Pygame' %}
+import pygame
+import sys
+import os
+
+try:
+    from importlib import metadata as importlib_metadata
+except ImportError:
+    # Backwards compatibility - importlib.metadata was added in Python 3.8
+    import importlib_metadata
+
+SCREEN_WIDTH, SCREEN_HEIGHT = 800, 600
+WHITE = (255, 255, 255)
+
+
+def main():
+    # Linux desktop environments use app's .desktop file to integrate the app
+    # to their application menus. The .desktop file of this app will include
+    # StartupWMClass key, set to app's formal name, which helps associate
+    # app's windows to its menu item.
+    #
+    # For association to work any windows of the app must have WMCLASS
+    # property set to match the value set in app's desktop file. For PPB this
+    # is set using environment variable.
+
+    # Find the name of the module that was used to start the app
+    app_module = sys.modules["__main__"].__package__
+    # Retrieve the app's metadata
+    metadata = importlib_metadata.metadata(app_module)
+
+    os.environ["SDL_VIDEO_X11_WMCLASS"] = metadata["Formal-Name"]
+
+    pygame.init()
+    pygame.display.set_caption(metadata["Formal-Name"])
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+
+    running = True
+    while running:
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                running = False
+                break
+
+        screen.fill(WHITE)
+        pygame.display.flip()
+
+    pygame.quit()
+    sys.exit()
```

Comment:
This line is a problem, for two reasons.

Firstly, sys.exit() with no arguments defaults to a return code of -1, which is an error state. At the very least, it should be `sys.exit(0)` to indicate a normal process exit.

Secondly - it isn't needed anyway. When the `main()` method exits, it will fall back to ``__main__``, and then exit cleanly. There's no need for an explicit `sys.exit()` unless you want to report an explicit exit condition.

Reply:
For some reason most pygame tutorials I've seen seems to include this, so I thought there must be a good reason for that, but I couldn't find out why they do it...
Anyway, I deleted this line.